docker-compose.prod.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.9'
```

Comment:
Why are you deleting the docker-compose version specification??

Reply:
Ohhhhh, that is what it was!!!!  I took it as an unused leftover python version block  :facepalm: 
Will add it!

Reply:
fixed!